features/step_definitions/helper_services.rb

```diff
@@ -364,6 +364,25 @@
   end
 end
 
+# test-client-pod is a pod that has web client tools (curl, etc)
+# this pod is only used by Network Edge team for routing test
+Given /^I have a test-client-pod in the#{OPT_QUOTED} project$/ do |project_name|
```

Comment:
We can integrate the new step into `I have a pod-for-ping in the#{OPT_QUOTED} project` as most of the steps are the same